### PR TITLE
2026 Add static image map source provider

### PR DESCRIPTION
**Pull request by ramtob**

### CODE REVIEW

File: src/app/@ansyn/plugins/openlayers/mapSourceProviders/open-layers-static-image-source-provider.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		});
+
+		layer = new ImageLayer({
+			source,
+			extent
+		});
+		return this.addFootprintToLayerPromise(Promise.resolve(layer), metaData);
+	}
+
+	getThumbnailName(overlay: IOverlay): string {
```
Comment (asafMasa) [inline src/app/@ansyn/plugins/openlayers/mapSourceProviders/open-layers-static-image-source-provider.ts]: remove

File: src/app/@ansyn/plugins/openlayers/mapSourceProviders/open-layers-static-image-source-provider.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		protected cacheService: CacheService,
+		protected imageryCommunicatorService: ImageryCommunicatorService,
+		@Inject(MAP_SOURCE_PROVIDERS_CONFIG) protected mapSourceProvidersConfig: IMapSourceProvidersConfig,
+		protected errorHandlerService: ErrorHandlerService,
+		protected http: HttpClient) {
+		super(cacheService, imageryCommunicatorService, mapSourceProvidersConfig);
+	}
+
+	createAsync(metaData: ICaseMapState): Promise<any> {
+		let layer;
```
Comment (asafMasa) [inline src/app/@ansyn/plugins/openlayers/mapSourceProviders/open-layers-static-image-source-provider.ts]: delete

File: src/app/@ansyn/plugins/openlayers/mapSourceProviders/open-layers-static-image-source-provider.ts (hunk 1 above)
Comment (ramtob) [reply src/app/@ansyn/plugins/openlayers/mapSourceProviders/open-layers-static-image-source-provider.ts]: done

File: src/app/@ansyn/plugins/openlayers/mapSourceProviders/open-layers-static-image-source-provider.ts (hunk 2 above)
Comment (ramtob) [reply src/app/@ansyn/plugins/openlayers/mapSourceProviders/open-layers-static-image-source-provider.ts]: done

**Pull request merged**